src/SFA.DAS.Assessors.Api/Controllers/TrainingCoursesController.cs

```diff
@@ -65,5 +65,49 @@ public async Task<IActionResult> GetStandardOptionsList()
 
             return Ok(model);
         }
+
+        [HttpGet]
+        [Route("active")]
+        public async Task<IActionResult> GetActiveList()
+        {
+            try
+            {
+                var queryResult = await _mediator.Send(new GetActiveTrainingCoursesQuery());
+
+                var model = new GetCourseListResponse
+                {
+                    Courses = queryResult.TrainingCourses.Select(c => (GetCourseListItem)c).ToList()
+                };
+
+                return Ok(model);
+            }
+            catch (Exception e)
+            {
+                _logger.LogError(e, "Error attempting to get list of training courses");
+                return BadRequest();
```

Comment:
Not sure if this is something you want to change, but we've been trying to change the response returned on exception to be an internal server error, instead of a BadRequest response

Reply:
I've updated to return internal server error